SECURITY.md

```diff
@@ -6,6 +6,25 @@ The Jaeger project provides community support only for last minor version: bug f
 
 Security fixes are given priority and might be enough to cause a new version to be released.
 
+### Security Patch Policy
+
+CVEs in Jaeger code will be patched in the newest Jaeger releases. 
+
+### Dependencies Lifecycle Policy
+
+Dependencies are evaluated before being introduced to ensure they:
+
+1) are actively maintained
+2) are maintained by trustworthy maintainers
+3) are licensed in a way not to impact the Jaeger license.
```

Comment:
can be more precise here - they must be using a license on CNCF allowlist https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md

Reply:
Should be good with latest push